influxdb_client/client/influxdb_client.py

```diff
@@ -83,14 +83,16 @@ def __exit__(self, exc_type, exc_value, traceback):
         self.close()
 
     @classmethod
-    def from_config_file(cls, config_file: str = "config.ini", debug=None, enable_gzip=False, **kwargs):
+    def from_config_file(cls, config_file: str = "config.ini", debug=None, enable_gzip=False,
+                         config_name: str = "influx2", **kwargs):
```

Comment:
Please use `kwargs` instead of new argument:
```suggestion
    def from_config_file(cls, config_file: str = "config.ini", debug=None, enable_gzip=False, **kwargs):
```